AdminUi/test/AdminUi.Tests.Integration/StepDefinitions/IndividualQuotaStepDefinitions.cs

```diff
@@ -24,6 +27,40 @@ public void GivenAnIdentity()
         _identityAddress = "id12Pbi7CgBHaFxge6uy1h6qUkedjQr8XHfm";
     }
 
+    [Given(@"an Identity i with an IndividualQuota q")]
+    public async Task GivenAnIdentityIWithAnIndividualQuotaQ()
+    {
+        _identityAddress = "id12Pbi7CgBHaFxge6uy1h6qUkedjQr8XHfm";
```

Comment:
Maybe we should save this address is a global constant (`TestData.Identities.IdentityA`)